sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/implementation/instrumentation/EventHubsTracer.java

```diff
@@ -0,0 +1,255 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Do we have any tests for this class?

Reply:
Yes, it's tested with existing tracing tests (that used to test old helper in core-amqp) plus new unit tests, plus new integration tests. The overall test coverage for this is 93%

Reply:
<img width="608" alt="image" src="https://user-images.githubusercontent.com/2347409/198731515-4ad695ab-07cd-4475-8e58-cc94b7e34f8d.png">